### ISSUE TITLE
TypeError: __vite_ssr_import_4__.default is not a function

### ISSUE THREAD

**Issue by DerYeger** [vitest-dev/vitest#352]
### Describe the bug

My test ran flawlessly locally and in GitHub Actions.

After adding the dependency `ts-deepmerge`, my tests still pass locally.
However, they fail in the CI pipeline with the error `TypeError: __vite_ssr_import_4__.default is not a function`.

For a full log of the issue, see https://github.com/DerYeger/d3-graph-controller/runs/4651590663?check_suite_focus=true.

### Reproduction

1. Install dependency `ts-deepmerge`.
2. Run tests in GitHub Actions workflow.

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor            
    Memory: 16.89 GB / 31.92 GB
  Binaries:
    Node: 16.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.15 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 7.20.3 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (96.0.1054.62)
    Internet Explorer: 11.0.22000.120
  npmPackages:
    @vitejs/plugin-vue: 2.0.1 => 2.0.1 
    vite: 2.7.7 => 2.7.7
    vitest: 0.0.118 => 0.0.118
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by DerYeger** [comment]
I managed to fix it by updating my `vite.config.ts` to include

```ts
import { defineConfig } from 'vite'

export default defineConfig({
  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore
  test: {
    include: ['test/**/*.test.ts'],
    deps: {
      inline: ['ts-deepmerge'],
    },
    environment: 'jsdom',
  },
})
```

In particular, `inline: ['ts-deepmerge']` war required.

**Comment by tomlla** [comment]
I have the same error with [givens](https://www.npmjs.com/package/givens) package.

**Comment by sheremet-va** [comment]
Should be fixed in [0.0.120](https://github.com/vitest-dev/vitest/releases/tag/v0.0.120)

**Comment by DerYeger** [comment]
I can confirm that 0.0.120 fixed the issue.
